caikit/runtime/client/remote_module_base.py

```diff
@@ -83,7 +84,9 @@ def __init__(
 
         # Configure GRPC variables and threading lock
         self._channel_lock = Lock()
-        self._conn_channel: Union[grpc.Channel, Session] = None
+        self._conn_channel: Optional[Union[grpc.Channel, Session]] = None
+        self._current_conn_time = None
+        self._max_conn_delta = timedelta(seconds=self._connection.max_session_age)
```

Comment:
Do we need a check on whether `max_session_age` is valid?

Reply:
Nope, that should be checked on `ConnectionInfo` creation

Reply:
Ok, yeah, with the removal of `Optional` I agree that this is fine